src/common/gui/RuntimeFont.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "vstgui/vstgui.h"
+#include "vstgui/lib/cfont.h"
+#include "vstgui/lib/platform/iplatformfont.h"
+
+namespace Surge
+{
+namespace GUI
+{
+
+/**
+ * initializeRuntimeFontFromDistribution
+ *
+ * The role of this function is to load a font from the bundle, dll, so whatever
+ * and plumb through the VSTGUI to create the 9 and 14 point fonts onto the two
+ * global variables "surge_minifont" and "surge_patchfont" if they are NULL
+ * at calltime.
+ *
+ * This is a remarkably OS-specific thing to do, so it is implemented in
+ * mac/win/linux RunTimeFontOS.cpp. (As of this writing it is neither implemented
+ * nor referenced on linux)
+ */
+void initializeRuntimeFontFromDistribution();
+
+}
+}
+
+/*
+** The two extern globals we need to initialize, which are defined/created
+** in SurgeGuieditor
+*/
+extern VSTGUI::CFontRef surge_minifont;
+extern VSTGUI::CFontRef surge_patchfont;
```

Comment:
I think we could rename stuff every once in a while when we do something else related. You could do a prepending commit for this that renames these as:

- displayFont (maybe?)
- patchFont

No reason to use surge prefix.

With Git renames are a breeze:

```
git grep -l get_modulation_f01|xargs sed -i -e 's/get_modulation_f01/modToFactor/g'
```

(Ignore the specific example. One day I was just thinking that clearer names for those would be just modToFactor and factorToMod, not part of any PR ATM)

Reply:
I am happy to do this but would sooner do it as a separate PR and commit. I prefer text-only changes to be commits on their own not intermixed with function changes (and I know you do too!) So once this is swept I'll do another PR renaming the variables.